packages/ipfs-http-client/src/version.js

```diff
@@ -1,19 +1,24 @@
 'use strict'
 
-const configure = require('./lib/configure')
 const toCamel = require('./lib/object-to-camel')
 
-module.exports = configure(({ ky }) => {
-  return async options => {
-    options = options || {}
+/** @typedef { import("./lib/api") } API */
 
-    const res = await ky.post('version', {
+/**
+ * Version
+ * @param {API} api
+ * @returns {function(Object): Promise<Object>}
+ */
+const version = (api) => {
+  return async (options = {}) => {
+    const res = await (await api.post('version', {
```

Comment:
The double await here - and also in places like `bitswap/wantlist` - would be more readable split out into separate lines

Reply:
@hugomrdias Curious for your thoughts on this when you get a chance. Thanks!

Reply:
i will fix this in another PR later